Leave/leave_db.py

```diff
@@ -29,12 +30,17 @@ def GetEmergencyLeavesForYear(member_id, year):
     return leaves
 
 def GetLeaveBalance(member_id, leave_type):
```

Comment:
Let's change this to `GetAnnualLeaveBalance` cause we shouldn't need to check for balance for other leave types

Reply:
Done